2001-3000/LIT2967RepCh248.xml

```diff
@@ -59,6 +59,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange unit="item">248</citedRange>
                </bibl>
             </listBibl>
+            <listRelation><relation name="betmas:formerlyAlsoListedAs" active="LIT2967RepCh248" passive="LIT3984basamay"/></listRelation>
```

Comment:
I was now wondering if also title (here and elsewhere) could be updated for better searchability
I mean

```
<title xml:id="t1" xml:lang="gez">በሰማይ፡ ወበምድር፡ አልብየ፡ ባዕደ።</title>
<title corresp="#t1" xml:lang="gez" type="normalized>Basamāy .....</title>
<title xml:id="t2">Hymn to Our Lady Mary</title>
<title xml:id="t3">Poem to Our Lady Mary</title>
```
Not necessary of course but might be helpful to quickly disambiguate in such cases, since the text is not on the record's first page in the view